### PR TITLE
Create public pipeline

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Create public pipeline to execute basic checks on project. Those are:

1. Execute Unit Tests
2. Execute lintin
3. Build Project

### CODE REVIEW

File: .github/workflows/build.yml
Comment (ghost) [inline .github/workflows/build.yml:6]: CodeQL needs that permission.

**Review by cpoile — APPROVED** [review]
Hey @pfltdv I'm not sure what this is for... I'll approve, but if you're looking for a serious dev-side review, we should ping someone else.

**Comment by ghost** [comment]
> Hey @pfltdv I'm not sure what this is for... I'll approve, but if you're looking for a serious dev-side review, we should ping someone else.

Hello @cpoile , thanks alot. We are testing Codeowners files in the repositories. So depending on the changes in PR, different teams can be notified. Cause of go.sum change in the repo, core developer's reviews is needed. Looks like we need to review that file again.

**Comment by agarciamontoro** [comment]
I also got a notification here and don't know either what this is :joy:

**Comment by ghost** [comment]
> I also got a notification here and don't know either what this is joy

Sorry @agarciamontoro, looks like we found a way to ping everyone :D.

**Comment by ghost** [comment]
Closing this PR, using groups at codeowners may not be good idea.

**Pull request closed without merging**
